src/core/tests/ov_tensor_test.cpp

```diff
@@ -76,6 +112,16 @@ TEST_F(OVTensorTest, canAccessF16Tensor) {
     EXPECT_THROW(t.data<std::int16_t>(), ov::Exception);
 }
 
+TEST_F(OVTensorTest, canAccessStringTensor) {
+    ov::Shape shape = {4, 3, 2};
+    ov::Tensor t{ov::element::string, shape};
+    EXPECT_NE(nullptr, t.data());
+    EXPECT_NO_THROW(t.data(ov::element::string));
+    EXPECT_NO_THROW(t.data<std::string>());
```

Comment:
what if I want to print such string tensor?
E.g. `t.data<std::string>()` and print all values

I suppose in ctor of string tensor you call `init` method and we can validate it via printing those values that were filled in `init()`

Reply:
added such check, thanks